Controllers/SubscriptionController.cs

```diff
@@ -0,0 +1,37 @@
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Tabloid.Data;
+using Tabloid.Models;
+
+
+[ApiController]
+[Route("/api/[controller]")]
+public class SubscriptionController : ControllerBase
+{
+    private TabloidDbContext _db;
+
+    public SubscriptionController(TabloidDbContext context)
+    {
+        _db = context;
+    }
+
+
+   [HttpPost]
+   [Authorize]
+   public IActionResult CreateSubscription(Subscription newSubscription)
```

Comment:
Consider making a DTO so only the properties the user should be able to set are exposed (`SubscriberId` and `FollowerId`). `SubbedDate` and `UnsubbedDate` shouldn't be exposed to the user as they are set by the endpoint, but more important is the Id, as the endpoint will fail if the Id is set to a value where there already exists a subscription with that Id.

Reply:
DTOs can also be used to represent which data the user can give. [Here ](https://ardalis.com/web-api-dto-considerations/) is an article about it, with a section on using DTOs to receive data under the Create header.